src/web/screens/keyPairsScreen.tsx

```diff
@@ -19,7 +19,8 @@ function KeyPairsScreen() {
     const data = await GetKeyPairs(participant!.id!);
     const sortedKeyPairs = data?.sort((a, b) => a.created.getTime() - b.created.getTime());
     setKeyPairData(sortedKeyPairs);
-  }, [participant]);
```

Comment:
This was causing an extra re-run of this method, but I don't think it's necessary. Let me know if you think otherwise

Reply:
I think there are two ways to solve it
1. make this call in loader, so this will only be called when it gets to the page
2. remove `participant!.id!` from GetKeyPairs, and use `participants/${participantId ?? 'current'}/keyPairs`. I am pretty sure this endpoint will go pass the enrichCurrentParticipant middleware

Reply:
and we could remove   
`const { participant } = useContext(ParticipantContext);`

Reply:
thanks! I went ahead and used your second suggestion to allow `GetKeyPairs` to be param-less. How's that look?

Reply:
LGTM

Reply:
the participant is used for `AddKeyPair`. Are you suggesting I do a similar treatment to that method and just make it not accept a `participantId`? So it will just be 
```
    const result = await axios.post(
      `/participants/current/keyPair/add`,
      props
    );
```

Reply:
or less drastically, just make `participantId` on `AddKeyPairFormProps` optional

Reply:
and have
```
    const result = await axios.post(
      `/participants/${participantId ?? 'current'}/keyPair/add`,
      props
    );
```

Reply:
I've done the latter

Reply:
Oh sorry, I didn't see participantId is using handleAddKeyPair. 
 I've noticed this `participant` changes after each rendering, I guess if it does not cause any rerender, we are fine with leave `const { participant } = useContext(ParticipantContext);`